masterlist.yaml

```diff
@@ -5012,6 +5012,78 @@ plugins:
   - name: 'Project Nevada - Med-X Fix.esp'
     tag: [ Names ]
 
+# Mothership Zeta Rehaul
+  - name: '(MothershipZetaRehaul|ZetaRehaul(BiWastelands|DelayDLCRedux|HardcoreZeta|HardcoreZetaDelayDLCRedux|NVSpeechChecks)Patch\.es[mp]'
```

Comment:
There is a closing `)` missing at the end, so: `Patch)\.es[mp]`

Also instead of `HardcoreZeta|HardcoreZetaDelayDLCRedux` you can write `HardcoreZeta(DelayDLCRedux)?`. The quantifier `?` means `once or none`.